test/test_contractiontreetograph.jl

```diff
@@ -0,0 +1,70 @@
+using ITensorNetworks
+using ITensorNetworks: create_contraction_tree, leaf_node, non_leaf_edges, edge_bipartition
+using Test
+using ITensors
+
+@testset "contractiontreetograph" begin
+
+    n =3
+    dims = (n,n)
+    g = named_grid(dims)
+    s = siteinds("S=1/2", g)
+
+    ψ = randomITensorNetwork(s; link_space=2)
+    ψψ = flatten_networks(ψ,ψ)
+
+    seq = contraction_sequence(ψψ);
+
+    g_seq = create_contraction_tree(seq)
+
+    #Get all leaf nodes (should match number of tensors in original network)
+    #why does vertices(g_seq)[leaf_node(g_seq, vertices(g_seq))) .== true] not work?"""
+    g_seq_leaves = vertices(g_seq)[findall(==(1), [leaf_node(g_seq,v) for v in vertices(g_seq)])]
```

Comment:
Isn't this already a function? It seems better to avoid complicated code like this in a test.